R/net.record.R

```diff
@@ -0,0 +1,105 @@
+#' @title Record time varying node values
+#'
+#' @description
+#' This function records values specific to a time step and a group of nodes.
+#' The nodes are identified by their \code{unique_ids} which allows the
+#' recording of data for nodes that are no longer in the network by the end of
+#' the run. The records are stored in \code{dat$node.records} and can be
+#' accessed from the \code{netsim} object with \code{get_node_records}
+#'
+#' @param dat a Master list object of network models
+#' @param at the time step where the recording happens
+#' @param measure the name of the value to record
+#' @param posit_ids a numeric vector of posit_ids to which the measure applies
+#' @param values the values to be recorded
+#'
+#' @return The Master list object
+#'
+#'
+#a Master list object of network models' @details
+#' See the "Time Varying Attributes" vignette
+#'
+#' @examples
+#' \dontrun{
```

Comment:
can be removed

Reply:
the don't run part ? 

I opened the issue we discussed the other day about that: https://github.com/statnet/EpiModel/issues/567